VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/camera-check/camera-check.component.html

```diff
@@ -5,19 +5,19 @@
 </div>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <h1 class="govuk-heading-xl">Was your camera working?</h1>
+    <h1 class="govuk-heading-xl">{{ 'camera-check.was-your-camera-working' | translate }}</h1>
 
     <form [formGroup]="form" class="govuk-form-group" (ngSubmit)="onSubmit()">
       <div [ngClass]="showError() ? 'govuk-form-group--error' : 'govuk-form-group'">
         <div *ngIf="showError()" class="govuk-error-message">
-          <span class="govuk-visually-hidden">Error:</span>
-          Select if you could see yourself in the camera window
+          <span class="govuk-visually-hidden">{{ 'camera-check.error' | translate }}</span>
+          {{ 'camera-check.select-if-you-could-see-yourself-in-the-camera-window' | translate }}
```

Comment:
could we summarise? maybe
`camera-check.could-see-yourself`

Reply:
done